java/core/src/java/org/apache/orc/impl/TreeReaderFactory.java

```diff
@@ -688,17 +700,23 @@ protected void skipRows(long items) throws IOException {
   }
 
   public static class DoubleTreeReader extends TreeReader {
-    protected InStream stream;
-    private final SerializationUtils utils;
+    private DoubleReader reader;
```

Comment:
Hi @pudidic , thank you for the patch!
I found `reader` should be `protected` here for Hive integration. [EncodedTreeReaderFactory](https://github.com/apache/hive/blob/master/ql/src/java/org/apache/hadoop/hive/ql/io/orc/encoded/EncodedTreeReaderFactory.java#L950)
And since we would remove a `protected` variable, could we reuse the `stream`?